packages/nextjs/components/ui/button.tsx

```diff
@@ -4,21 +4,21 @@ import { type VariantProps, cva } from "class-variance-authority";
 import { cn } from "~~/lib/utils";
 
 const buttonVariants = cva(
-  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50",
+  "inline-flex items-center justify-center whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-0 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50",
```

Comment:
notice ring-0: With ring-1 behavior is [strange](https://github.com/radix-ui/primitives/issues/1164), inside dropdown it's focused even on hover and you cant move focus with `tab` (but can with `up` and `down` buttons). And after page change there's no focus on hover. Haven't met radixui bugs before though. So for now I set it to ring-0

https://github.com/user-attachments/assets/b1c9150a-6a1c-4bae-8723-905b21fc4ba2

Update: I think it's ok to make it the same color as button. Also found a small bug that our menu doesn't open on focus + click to enter. Will fix it tomorrow

Reply:
updated ring color to ring-accent for now since same color as button makes it invisible. Fixed menu and qrcode modal so they can be opened by `enter` press and closed by `esc`. Other menu buttons also works on `enter` except `Copy address` because `CopyToClipboard` component works only on click. Looks like it's better to write our custom component, since `react-copy-to-clipboard` is outdated, written as [class comonent](https://github.com/nkbt/react-copy-to-clipboard/blob/master/src/Component.js) and needs additional types https://github.com/scaffold-eth/scaffold-eth-2/pull/933#issue-2515987545. If you agree, I'll create an issue for it